SPECS-EXTENDED/freeradius/freeradius.spec

```diff
@@ -1,11 +1,12 @@
 %global _default_patch_fuzz 2
 # Is elliptic curve cryptography supported?
 %global HAVE_EC_CRYPTO 1
+%global debug_package %{nil}
```

Comment:
Why are we removing the debug package ?

Reply:
The debug symbols it is generating is conflicting with debug symbols from filesystem package. Any suggestions on how to resolve it in a way without not generating debug info sub-package?

Reply:
I am not aware of how could we resolve this.
I would request creating a bug though so that we do not loose track of it.
There is a precedent where we disable debug packages.

Reply:
done.